app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Set/Grid.php

```diff
@@ -52,6 +54,10 @@ protected function _prepareCollection()
         return parent::_prepareCollection();
     }
 
+    /**
+     * @return void
```

Comment:
Just curious, why `@return void` is not useless here?

Reply:
This method has no return statement, but the parrent method has one.

(adding `return $this;` would be possible too - maybe in another PR)

Reply:
Thanks, it makes sense now. I'm learning so much new stuff.